### PR TITLE
Update helpdesk

### PR DESCRIPTION
**Pull request by osuShiro**
Adding a few updates for features that were added since last one

### CODE REVIEW

File: wiki/Help_Centre/en.md [hunk 1]
```diff
@@ -110,17 +110,17 @@ We only grant this one time per account, so please consider carefully before req
...
 
 If the change was done with the free change that comes with a supporter tag, we cannot offer to change it again.
 
 #### Can I remove my previous usernames/delete username history?
 
-**Not at the moment, sorry.**
+**Under certain circumstances.**
 
-Your profile will show all your previous purchased namechanges, and we do not accept request to remove them for now. If one of your previous usernames breaches your privacy (for example, if it contains your legal name), contact accounts@ppy.sh to request it removed.
+Your profile will show all your previous purchased namechanges, but we only selective remove the ones that may be an issue for you. If one of your previous usernames breaches your privacy (for example, if it contains your legal name) or can lead to your privacy being breached (via other websites), contact accounts@ppy.sh to request it removed.
```
Comment (TicClick) [inline wiki/Help_Centre/en.md]: `selective` -> `selectively`

File: wiki/Help_Centre/en.md [hunk 2]
```diff
@@ -626,7 +628,7 @@ Once all of the `.osz` files within the pack are in the Songs directory, hitting
 
 **The vast majority of the time, this is due to a high number of tickets coming into our system at once.**
 
-You can always send a single reply to your currently-open ticket asking about the state of your case, but make sure not to do this more than once or twice a week.
+You can always send a single reply to your currently-open ticket asking about the state of your case, but make sure not to do this more than once or twice a week. You can also contact us on twitter at @osusupport to ask about your ticket.
```
Comment (TicClick) [inline wiki/Help_Centre/en.md]: `currently-open` -> `currently open`, no?

File: wiki/Help_Centre/en.md [hunk 3]
```diff
@@ -658,6 +660,8 @@ In the event that another user is harassing you via private messages either on t
 
 You'll no longer see public chat messages from any users on that list, or receive private messages from them.
 
+If you wish to ignore highlights, private messages, or public messages, you can simply append respectively @h, @p, @c to their username. `The_Gatekeeper@c` will ignore the user's messages in public channels. It is possible to combine those: `The_Gatekeeper@ph` will ignore highlights and private messages, but leave public messages visible.
```
Comment (TicClick) [inline wiki/Help_Centre/en.md]: `@h`, `@p`, `@c`

File: wiki/Help_Centre/en.md [hunk 4]
```diff
@@ -687,9 +691,11 @@ If you wish to prevent anyone not on your friendslist from sending you private m
 6. Click the first dropdown box once more and change its value to "Delete message", then click "Add Rule".
 7. You are done! Any attempts the user makes to contact you via forum PMs will now be automatically deleted without warning to you.
 
+On the new website, go to your [account settings](https://osu.ppy.sh/home/account/edit) and simply click _lock private messages from people not on your friends list_ in the Privacy section. Private messages from users in your friendlist will still reach you.
```
Comment (TicClick) [inline wiki/Help_Centre/en.md]: for interface labels, I would use code blocks as well: `lock private messages from people not on your friends list`

File: wiki/Help_Centre/en.md [hunk 5]
```diff
@@ -170,7 +170,9 @@ If you are playing from a public place, make sure to keep a close eye on your ac
 
 **You can see all maps you've played on the website.**
 
-If you have an active supporter tag, you can see all maps you have submitted a score on by going in the Beatmap section, then clicking the "Ranked (played)" filter. Alternatively, you can use [this link](https://osu.ppy.sh/p/beatmaplist?m=-1&r=7&g=0&la=0&ra=) to access the page directly.
+**If you have an active supporter tag**, you can see all maps you have submitted a score on by going in the Beatmap section, then clicking the "Ranked (played)" filter. Alternatively, you can use [this link](https://osu.ppy.sh/p/beatmaplist?m=-1&r=7&g=0&la=0&ra=) to access the filtered page directly.
```
[the remark this answers is not in the document]
Comment (osuShiro) [reply wiki/Help_Centre/en.md]: Could you list them here? I'll fix all of them in this PR while I'm at it

File: wiki/Help_Centre/en.md (hunk 2 above)
Comment (osuShiro) [reply wiki/Help_Centre/en.md]: currently-open is technically correct (and would actually be better in this instance) but i'll remove the hyphen to make it simpler

Comment (cl8n) [reply wiki/Help_Centre/en.md]: most style guides recommend using hyphens for compound adjectives, *except* where the first word is the adverb "very" or an adverb ending in "ly"

so `currently open` is technically correct too =) one of the many dumb gotcha's of english

File: wiki/Help_Centre/en.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/Help_Centre/en.md]: - password recovery / "forgot" links should be https://osu.ppy.sh/home/password-reset
- stat.ppy.sh => status.ppy.sh
- UCP stuff under "Blocking public forum posts from a user" and "Prevent any new forum private messages from reaching you" is no longer available, and there's another mention of "new website" after it

File: wiki/Help_Centre/en.md [hunk 6]
```diff
@@ -626,7 +628,7 @@ Once all of the `.osz` files within the pack are in the Songs directory, hitting
 
 **The vast majority of the time, this is due to a high number of tickets coming into our system at once.**
 
-You can always send a single reply to your currently-open ticket asking about the state of your case, but make sure not to do this more than once or twice a week.
+You can always send a single reply to your currently open ticket asking about the state of your case, but make sure not to do this more than once or twice a week. You can also contact us on twitter at @osusupport to ask about your ticket.
```
Comment (cl8n) [inline wiki/Help_Centre/en.md]:
```suggestion
You can always send a single reply to your currently open ticket asking about the state of your case, but make sure not to do this more than once or twice a week. You can also contact us on Twitter at [@osusupport](https://twitter.com/osusupport) to ask about your ticket.
```

**Pull request merged**